### PR TITLE
Update skuba to crio v1.18rc1

### PR DESCRIPTION
**Pull request by mjura**
## Why is this PR needed?

It is upgrading `cri-o` to `v1.18rc1` version

## What does this PR do?

New version `cri-o v1.18rc1`

# Merge restrictions

(Please do not edit this)

We are in *v4-maintenance phase*, so we will restrict what can be merged to prevent unexpected surprises:

    What can be merged (merge criteria):
        2 approvals:
            1 developer: code is fine
            1 QA: QA is fine
        there is a PR for updating documentation (or a statement that this is not needed)

### CODE REVIEW

**Comment by evrardjp** [comment]
I am not sure yet how this applies for upgrades. I have to investigate before giving my final review.

**Comment by evrardjp** [comment]
Please note that due to the urgency of the fix (which is quite relative), I might change my opinion to merge it asap when things have been clarified.

**Comment by mjura** [comment]
@evrardjp Issue with parsing string capabilities flag will be fixed in upstream for cri-o v1.18. Upgrade path will be covered in https://github.com/SUSE/avant-garde/issues/1513 
First we would like to fix CI for `Devel:CaaSP:4.0`

File: pkg/skuba/actions/cluster/init/manifests.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ const (
 ## Default        : ""
 ## ServiceRestart : crio
 #
-CRIO_OPTIONS=--pause-image={{.PauseImage}}{{if not .StrictCapDefaults}} --default-capabilities="CHOWN,DAC_OVERRIDE,FSETID,FOWNER,NET_RAW,SETGID,SETUID,SETPCAP,NET_BIND_SERVICE,SYS_CHROOT,KILL,MKNOD,AUDIT_WRITE,SETFCAP"{{end}}
+### BUG [ CRIO v1.18.0rc1 ] - string parsing issue based on comma separators
```
Comment (evrardjp) [inline pkg/skuba/actions/cluster/init/manifests.go:27]: I was told by @saschagrunert that the new parsing is how it is intended to be, so it means that if we keep the `not StrictCapDefault` feature (which we are supposed to, as it's bound to caasp4 lifetime), we'll need to update this anyway for old provisioned clusters.

**Comment by evrardjp** [comment]
> @evrardjp Issue with parsing string capabilities flag will be fixed in upstream for cri-o v1.18. Upgrade path will be covered in [SUSE/avant-garde#1513](https://github.com/SUSE/avant-garde/issues/1513)
> First we would like to fix CI for `Devel:CaaSP:4.0`

So I am told. But I am also told that _this is the intended behaviour_, which makes me think we can have regressions in the future as long as we don't adapt. @saschagrunert can you confirm?

**Comment by mjura** [comment]
> Yeah, I understand that. Let's just be clear into how we proceed.

Merge this change `Devel:CaaSP:4.0` and unblock CI then prepare migration path in follow-up PR's.

**Comment by mjura** [comment]
It is temporary fix for crio-v1.18rc1 testing purpose. Full fix and migration path will be covered in https://github.com/SUSE/avant-garde/issues/1513

**Pull request merged**